webui/react/src/components/Table/Table.tsx

```diff
@@ -83,7 +83,7 @@ export const HumanReadableNumberRenderer = (num: number): React.ReactNode => {
 export const relativeTimeRenderer = (date: Date): React.ReactNode => {
   return (
     <Tooltip title={date?.toLocaleString()}>
```

Comment:
seems like this `Tooltip` is unnecessary., but i leave it alone for now

Reply:
Asked Emily (code author) about this, and it's ok to remove this tooltip cuz `TimeAgo` already have one